django/thunderstore/github/views.py

```diff
@@ -0,0 +1,54 @@
+import base64
+import json
+from typing import Any
+
+from cryptography.hazmat.primitives.asymmetric.ec import ECDSA
+from cryptography.hazmat.primitives.hashes import SHA256
+from cryptography.hazmat.primitives.serialization import load_der_public_key
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from thunderstore.account.models import ServiceAccount
+from thunderstore.account.tokens import hash_service_account_api_token
+from thunderstore.github.models.keys import KeyProvider, KeyType
+from thunderstore.github.utils import solve_key, unpem
+
+
+class SecretScanningEndpoint(APIView):
+    """
+    Endpoint for receiving Githubs secret scanning matches
+    """
+
+    def __init__(self, **kwargs: Any) -> None:
+        # Exclude this from the API documentation
+        self.schema = None
```

Comment:
Do we want to add this to the DRF documentation. I just removed it since, I figured this wasn't going to be a "listed" public endpoint, what I mean is that we don't want to advertise this endpoint🤔